code/datums/traits/negative.dm

```diff
@@ -100,94 +100,339 @@
+						   /obj/item/radio/off)
+	if(sci || prob(1))
+		heirloom_table.Add(/obj/item/toy/plush/slimeplushie,
+						   /obj/item/reagent_containers/food/snacks/monkeycube,
+						   /obj/item/screwdriver,
+						   /obj/item/wrench,
+						   /obj/item/multitool)
+	if(med || prob(1))
+		heirloom_table.Add(/obj/item/clothing/neck/stethoscope,
+						   /obj/item/book/manual/wiki/medicine,
+						   /obj/item/bodybag,
+						   /obj/item/surgical_drapes,
+						   /obj/item/scalpel,
+						   /obj/item/hemostat,
+						   /obj/item/retractor,
+						   /obj/item/cautery,
+						   /obj/item/bedsheet/medical)
+	if(engi || prob(1))
+		heirloom_table.Add(/obj/item/screwdriver,
+						   /obj/item/wrench,
+						   /obj/item/weldingtool,
+						   /obj/item/crowbar,
+						   /obj/item/wirecutters)
+	if(supp || prob(1))
+		heirloom_table.Add(/obj/item/hand_labeler,
+						   /obj/item/shovel)
+	if(comm || prob(1))
+		heirloom_table.Add(/obj/item/reagent_containers/food/drinks/flask/gold,
+						   /obj/item/book/manual/wiki/security_space_law,
+						   /obj/item/clothing/glasses/sunglasses/advanced/gar/supergar,
+						   /obj/item/stamp,
+						   /obj/item/stamp/denied)
+
+	// 4-1. For everyone
+	heirloom_table.Add(pick(subtypesof(/obj/item/bedsheet)), //random bedsheet. you can get a fancy one if you're lucky.
+					   /obj/item/toy/cards/deck,
+					   /obj/item/lighter,
+					   /obj/item/dice/d20,
+					   /obj/item/book/manual/wiki/security_space_law) //1984. all crews are encourage to hold this book all times. giving higher chance for sec.
+
+	// 4-2. Rare chance to be fucked up
+	if(prob(5)) //with 5% chance, these items are added to your table - which means you still have a chance to avoid them.
+		if(comm) //get one of duffelbags
+			heirloom_table.Add(/obj/item/storage/backpack/duffelbag/captain)
+		else if(sec)
+			heirloom_table.Add(/obj/item/storage/backpack/duffelbag/med)
+		else if(med)
+			heirloom_table.Add(/obj/item/storage/backpack/duffelbag/sec)
+		else if(engi)
+			heirloom_table.Add(/obj/item/storage/backpack/duffelbag/engineering)
+		else if(sci)
+			heirloom_table.Add(/obj/item/storage/backpack/duffelbag/engineering) //there's no sci theme duffelbag
+		else if(quirk_holder.mind.assigned_role == "Clown")
+			heirloom_table.Add(/obj/item/storage/backpack/duffelbag/clown)
+		else
+			heirloom_table.Add(/obj/item/storage/backpack/duffelbag)
+		heirloom_table.Add(/obj/item/storage/backpack/duffelbag/syndie, 	//and another chance to get syndi one. woo, even suspicious.
+						   /obj/item/skub) 		//more worse
+		if(!comm && !sec) //crews will suspiciously get this traitor operative item, but they can't do anything with this. security will just be upset at you and taking your heirloom.
+			heirloom_table.Add(/obj/item/nuke_core_container/supermatter)
+		else
+			heirloom_table.Add(/obj/item/storage/backpack/duffelbag) //just balancing for chance to opposite departments
```

Comment:
no

Reply:
just dislike the idea?

Reply:
giving out antagonist items doesn't sound like a good idea to me